core-blocks/quote/editor.scss

```diff
@@ -1,12 +1,13 @@
-.wp-block-quote {
+.editor-block-list__block[data-type="core/image"] .wp-block-quote {
```

Comment:
It looks like a bug here `"core/image"`? Also, be mindful that we should try to avoid these selectors because the `.editor-block-list__block[data-type="core/*"]`div is not rendered when you save the block as reusable breaking the styles when editing reusable blocks.

Reply:
Definitely a bug. Fixed typo, thanks. 

> Also, be mindful that we should try to avoid these selectors because the .editor-block-list__block[data-type="core/*"]div is not rendered when you save the block as reusable breaking the styles when editing reusable blocks.

Right — this would be for scoping _editor-only_ styles. Any style that should be shown in both the editor and the front-end should presumably be in style.scss right?

Reply:
I think Riad is trying to explain that when you save your block as reusable (Saved) block, those rules won't be applied, because it is going to be controlled by `data-type="data"` matcher.